src/chronos/chronos.py

```diff
@@ -209,6 +209,37 @@ def __init__(self, config: ChronosConfig, model: PreTrainedModel) -> None:
     def device(self):
         return self.model.device
 
+    @torch.no_grad()
```

Comment:
I'm wondering: should this be enforced here, or rather at invocation time?

Reply:
Moved to `pipeline.embed` instead.